.gitignore

```diff
@@ -6,3 +6,14 @@ target
 **/*.rs.bk
 node_modules
 test-ledger
+xNFT_Ansh
+xNFT_Default
+xNFT_Tim
+ticket-xnft
+client
+Cargo.lock
+package-lock.json
+package.json
+yarn.lock
+
+
```

Comment:
Why are they included in `.gitignore` file?

Reply:
fixed in next commit